src/main.c

```diff
@@ -0,0 +1,110 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include "math.h"
+#include "debug.h"
+#include "helpers.h"
+
+
+int main(int argc, char *argv[]) {
+    int opt, num1, num2 = 0;            // Only works for integers for now
+    char operation = 0;             // Which operator we are going to use
+    int has_numbers = 0;            // Status: Does *argv have numbers?
+
+    // Parse command-line options
+    while ((opt = getopt(argc, argv, "a:s:m:d:hv")) != -1) // CMD Line options 
+    {
+        switch (opt) 
+        {
+            case 'a': // Add
+                operation = 'a';
+                num1 = atof(optarg);
```

Comment:
1. You can assume these will be integer not float values

2. Use strtol rather than atoi